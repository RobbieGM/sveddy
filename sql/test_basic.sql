CREATE EXTENSION sveddy;
SELECT add_integers(5, 3);
