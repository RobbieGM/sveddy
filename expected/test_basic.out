CREATE EXTENSION sveddy;
SELECT add_integers(5, 3);
 add_integers 
--------------
            8
(1 row)

